config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
```

Comment:
Ok, I don't think this should be part of the config.  The high levels will provide a way for the user to specify what runtime it should be launched with.  

This should be safe to remove.